### DISCUSSION TITLE
Are you aware that the docker image for the cli does not have a "latest" tag?

### DISCUSSION THREAD

**Discussion by brassel** [geldata/gel#1862] [Q&A] [answered]
So, when people use the provided pull comand at https://hub.docker.com/r/edgedb/edgedb-cli

```
docker pull edgedb/edgedb-cli
```

they get an error

```
Error response from daemon: manifest for edgedb/edgedb-cli:latest not found: manifest unknown: manifest unknown
```

Might be worthwhile to add it. ;-)

**Comment by elprans** [comment, marked as answer]
This is somewhat intentional, because of multi-platform, multi-architecture concerns.  The tag you are looking for is `linux-x86_64-latest`.  

IIRC, it should be possible to have a "smart" `latest` tag via `docker manifest`, but we didn't get to that yet.

**Comment by brassel** [comment]
Thanks for the answer! By now I have found out that i can use the cli embedded in the main docker image as well.
